coordinatorlib/src/main/scala/com/socrata/datacoordinator/common/soql/sqlreps/UrlRep.scala

```diff
@@ -26,6 +28,52 @@ class UrlRep(val base: String) extends RepUtils with SqlColumnRep[SoQLType, SoQL
     s"${physColumns(urlOffset)}=?",
     s"${physColumns(descriptionOffset)}=?").mkString(",")
 
+
+  def sql_==(literal: SoQLValue): String = {
+    val url = literal.asInstanceOf[SoQLUrl]
+    url match {
+      case SoQLUrl(Some(url), Some(description)) =>
+        s"(${base}_url = ${sqlescape(url)} AND ${base}_description = ${sqlescape(description)})"
+      case SoQLUrl(Some(url), None) =>
+        s"(${base}_url = ${sqlescape(url)})"
+      case SoQLUrl(None, Some(description)) =>
+        s"(${base}_description = ${sqlescape(description)})"
+      case SoQLUrl(None, None) =>
+        s"(${base}_url is null and ${base}_description is null)"
```

Comment:
It seems weird to have this one case care about subcolumns being null, and not the other two.

Reply:
It's also weird that this is incongruent with `sql_in`'s behavior.

Reply:
fixed